pkg/client/cache/listers.go

```diff
@@ -52,6 +52,15 @@ func (s *StoreToPodLister) List(selector labels.Selector) (pods []api.Pod, err e
 	return pods, nil
 }
 
+// Returns true if a pod matching the namespace/name of the given pod exists in the store.
+func (s *StoreToPodLister) Exists(pod *api.Pod) bool {
```

Comment:
I thought you said the linter barfs if this isn't phrased as "StoreToPodLister returns true if ..." ? (BTW I think that policy is overly verbose and presumably optimizing for the uncommon case, which is that people are reading the godoc-processed code rather than the source code)

On a more concrete note, I'm not sure how I feel about this function converting the error into "false." It would probably be better if you propagated it up to the caller by returning bool, err.